pkg/e2e/framework/framework.go

```diff
@@ -236,14 +275,22 @@ func WaitUntilDeleted(delFnc func() error, getFnc func() error) error {
 	})
 }
 
+func (f *Framework) DefaultErrNotExpected(err error) {
```

Comment:
Why is it `Default` when we pass parameter?

Reply:
Since `Expect` exists in ginkgo/gomega. I am reusing the e2e framework in `libvirt-actuator bootstrap` (about to open PR for it) and implementing different error handling.